java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -630,6 +634,34 @@ public CompletableFuture<String[]> lpopCount(@NonNull String key, long count) {
                 response -> castArray(handleArrayResponse(response), String.class));
     }
 
+    @Override
+    public CompletableFuture<Long> lpos(@NonNull String key, @NonNull String element) {
+        return commandManager.submitNewCommand(
+            LPos, new String[] {key, element}, this::handleLongResponse);
+    }
+
+    @Override
+    public CompletableFuture<Long> lpos(@NonNull String key, @NonNull String element, @NonNull LPosOptions options) {
+        return commandManager.submitNewCommand(
+            LPos, new String[] {key, element, Arrays.toString(options.toArgs())}, this::handleLongResponse);
+    }
+
+    @Override
+    public CompletableFuture<Long[]> lposCount(@NonNull String key, @NonNull String element, long count) {
+        return commandManager.submitNewCommand(
+            LPos,
+            new String[] {key, element, Long.toString(count)},
```

Comment:
don't forget, you will need the keyword "COUNT" in front of the count. 
```suggestion
            new String[] {key, element, COUNT_REDIS_API, Long.toString(count)},
```
Except you should add COUNT_REDIS_API to the LPosOptions as a final static string COUNT_REDIS_API = "COUNT";

Reply:
example: https://github.com/aws/glide-for-redis/pull/1524/files#diff-6118fb6c768818f92c96d93eeee2f53e0d8e21a5929a030203034b1da383460aR20